pkg/kube/fake/printer.go

```diff
@@ -47,7 +48,11 @@ func (p *PrintingKubeClient) Create(resources kube.ResourceList) (*kube.Result,
 	return &kube.Result{Created: resources}, nil
 }
 
-func (p *PrintingKubeClient) Wait(resources kube.ResourceList, _ time.Duration) error {
+func (p *PrintingKubeClient) Wait(resources kube.ResourceList, d time.Duration) error {
+	if p.WaitDuration != 0 {
+		time.Sleep(p.WaitDuration)
```

Comment:
I think if I recall correctly, the time.Duration was meant more as a deadline (wait for X seconds) instead of calling `.Sleep`. So you'd set a `context.WithDeadline` and return an error if the deadline was reached.

https://golang.org/pkg/context/#WithDeadline

By introducing this, users who previously introduced a wait time duration may now have their libraries/projects sleep for the described duration, and I don't think that's how this field was intended to be used.

Reply:
From what I understood of the PrintingKubeClient's code, the time.Duration` doesn't do anything it is just a placeholder to make this function have the same signature that the real function of KubeClient